src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -280,7 +281,19 @@ function WorkspaceCategoriesPage({policy, policyCategories, route}: WorkspaceCat
                         />
                         {isSmallScreenWidth && <View style={[styles.pl5, styles.pr5]}>{getHeaderButtons()}</View>}
                         <View style={[styles.ph5, styles.pb5, styles.pt3]}>
-                            <Text style={[styles.textNormal, styles.colorMuted]}>{translate('workspace.categories.subtitle')}</Text>
+                            {Object.keys(policy?.connections ?? {}).length ? (
```

Comment:
```suggestion
                            {Object.keys(policy?.connections ?? {}).length > 0 ? (
```
NAB. Conditions should be boolean